lib/elixir/lib/process.ex

```diff
@@ -403,21 +403,21 @@ defmodule Process do
   end
 
   @doc """
-  Associates the name with a pid or a port identifier. `name`, which must
-  be an atom, can be used instead of the pid / port identifier with the
-  `Kernel.send/2` function.
+  Associates the atom `name` with a `pid` or a port identifier. `name`, can then
```

Comment:
Btw, the first paragraph of documentation should always be a quick and short summary. In this case, simply: "Associates the atom `name` with a pid or a port identifier."

Since you are already improving the docs, could you please also follow this convention? Thank you.